compiler/damlc/tests/visual/Basic.dot

```diff
@@ -2,30 +2,34 @@ digraph G {
 compound=true;
 rankdir=LR;
 subgraph cluster_Group{
-n1[label=Group_Archive][color=red]; 
+n10[label=Archive][color=red]; 
```

Comment:
This seems to confirm my suspicion that we now have `Archive` being represented twice.

Reply:
Ah good catch. will fix it thanks